charts/hermes-relayer/Chart.yaml

```diff
@@ -3,5 +3,5 @@ apiVersion: v2
 name: hermes-relayer
 description: A Helm chart for Kubernetes
 type: application
-version: 1.3.1
+version: 1.3.2
 appVersion: 1.5.1
```

Comment:
Lets please update the default here

Reply:
https://github.com/archway-network/helm-charts/pull/11/commits/aa2f2d108289e61192dda5ed5e64bf631e008e74